game-headed/build.install4j

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<install4j version="7.0.8" transformSequenceNumber="7">
+<install4j version="7.0.9" transformSequenceNumber="7">
```

Comment:
We shouldn't do that (I know it was done automatically) until we update the install4j file.
@ssoloff Can probably tell you more about this.

Reply:
See #4369 and triplea-game/assets#22 for what is required to upgrade install4j.  (If upgrading, the 7.0.8 installer can be deleted since it's not used by any other branch.)  That should probably be done as a separate PR.

Or you could just install 7.0.8 locally to use the correct IDE version when editing the build script.

Reply:
Thanks for the feedback. Considering it, I had the latest install4j installer file handy and it was pretty easy to get it uploaded to 'assets'.

The travis piece is updated to use 7.0.9 to match: f3e28a4

Reply:
The install4j Gradle plugin should also be updated to 7.0.9 (see #4369).

Reply:
I agree it is good house-keeping to keep that in-sync  :+1: 
20ad25de148b9c02effdf15d9784e16d97a57404